docs/UserGuide.md

```diff
@@ -524,7 +549,6 @@ For detailed explanations of each flag and acceptable arguments, refer to Sectio
 delete <INDEX>
 ```
 * Mandatory Field: `<INDEX>`
-* Note: The provided `<INDEX>` must be **greater than 0 and less than the total number of clients in the list**.
```

Comment:
Was this removed by accident? It seems quite important.

Reply:
This was centralised to the index arguments in Section 4, but I'll add a hyperlink as a quick reference!

Reply:
I think a hyperlink sounds good, but I feel repeating this is ok since its just a 1 line reminder

Reply:
Yes I second adding a hyperlink to the initial definition, to avoid repeating.

Reply:
Maybe we can briefly write it and have it linked to the section with more details!